dpbench/infrastructure/benchmark.py

```diff
@@ -54,9 +56,25 @@ def _setup_func(initialized_data, array_args, framework):
     return copied_args
 
 
+def _array_size(array: Any) -> int:
+    if isinstance(array, np.ndarray):
```

Comment:
np.ndarray and dpnp.ndarray can be combined in same block. dpnp will always have same API as NumPy.

Reply:
`AttributeError: 'dpnp_array' object has no attribute 'nbytes'`

Reply:
We can try something like:
```
try:
	return array.nbytes
except AttributeError:
	return array.size * array.itemsize 
```
And probably also file an issue for dndp to support nbytes